app/SentryServiceProvider.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Silex;
+
+use Sentry\ClientBuilder;
+use Sentry\State\Hub;
+use Sentry\State\HubInterface;
+use Silex\Application;
+use Silex\ServiceProviderInterface;
+use Throwable;
+
+class SentryServiceProvider implements ServiceProviderInterface
+{
+    public function register(Application $app)
+    {
+        $app['sentry_hub'] = $app->share(
```

Comment:
Nit: We could use the `HubInterface::class` constant instead. We mostly used "human-readable" names in the past becaus we didn't have the `::class` constant yet back then. We've started using it recently for service definitions in some repositories. Might as well start here as well in preparation for our move away from Silex in this app as well.

Reply:
That's indeed a good tip, I changed it! Also a good one to remember when doing some boyscouting.